### PR TITLE
Consolidate Test Visibility Doc Pages Into Setup

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Customizes the multi-lang-code parameters to add JavaScript and Swift, moves the Information Collected shortcode into the Test Visibility doc page, and consolidates eight pages into one Setup doc page in the Test Visibility section.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: layouts/partials/continuous_integration/ci-tests-setup.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <a class="card h-100" href="/continuous_integration/tests/setup/swift/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/swift_avatar.svg" "class" "img-fluid" "alt" "swift" "width" "200") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/continuous_integration/tests/setup/junit_xml/">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/junit.png" "class" "img-fluid" "alt" "upload junit tests" "width" "200") }}
```
Comment (juan-fernandez) [inline layouts/partials/continuous_integration/ci-tests-setup.html:50]: This is likely going to be confused by the java test framework, which is not quite what we want: many test frameworks for languages different than Java can create junit xml reports, but that's not the framework. We had a similar discussion when doing the onboarding page and we left the JUnit XML option without a logo: https://app.datadoghq.com/ci/setup/test?language=junit-xml

**Review by juan-fernandez — COMMENTED** [review]
LGTM but let's figure out if we want to use that JUnit logo

**Review by juan-fernandez — APPROVED** [review]

**Comment by maycmlee** [comment]
Created DOCS-6340 for review.

**Review by KickingTheTV — APPROVED** [review]
LGTM

**Review by romainkomorndatadog — APPROVED** [review]

**Pull request merged**